azure-servicebus/tests/control_plane_tests/test_servicebus_eventhub.py

```diff
@@ -61,6 +61,7 @@ def _create_event_hub(self, hub_name):
         self.sbs.create_event_hub(hub_name, None, True)
 
     #--Test cases for event hubs ----------------------------------------------
+
```

Comment:
It's not a _huge_ deal, but I'd pref that you leave the file as-is instead of introducing extra newlines.